composer.json

```diff
@@ -6,12 +6,13 @@
         "ruafozy/mersenne-twister": "^1.3"
     },
     "require-dev": {
-        "phpdocumentor/phpdocumentor": "^2.8.5"
+        "phpdocumentor/phpdocumentor": "^2.8.5",
+        "phpunit/phpunit": "~4.8.20"
```

Comment:
Using this, it is more clear on how tests are to be run; using composer will install phpunit